chain/cosmos/tx.go

```diff
@@ -30,17 +30,21 @@ const (
 	DefaultChainID = pack.String("testnet")
 	// DefaultSignMode used in signing the tx
 	DefaultSignMode = 1
+	// DefaultDecimals used in calculating gas price
+	DefaultDecimals = 1
```

Comment:
Maybe this should be called `DefaultDecimalsDivisor` and the comment updated to make it clear how this value is used (could include an example).